server/test/suite/dotConfig.test.mjs

```diff
@@ -23,31 +23,6 @@
 import { dotConfig } from "../../src/dotConfig.mjs";
 
 test(`dotConfig works properly`, () => {
-  const expectedResult = {
-    supportedSdks: [
-      {
-        cliFlag: null,
-        cliShortFlag: null,
-        enabled: true,
-        fileName: "firebolt-open-rpc.json",
-        name: "core",
-      },
-      {
-        cliFlag: "manage",
-        cliShortFlag: "m",
-        enabled: false,
-        fileName: "firebolt-manage-open-rpc.json",
-        name: "manage",
-      },
-      {
-        cliFlag: "discovery",
-        cliShortFlag: "d",
-        enabled: false,
-        fileName: "firebolt-discovery-open-rpc.json",
-        name: "discovery",
-      },
-    ],
-    validate: [ "method", "params", "response", "events" ],
-  };
-  expect(dotConfig).toEqual(expectedResult);
+  const result = JSON.parse(JSON.stringify(dotConfig));
```

Comment:
It seems like you are testing that the dotConfig object within the dotConfig module is a valid object. Not sure that's valuable. The code that loads the dotConfig file already has a try-catch when it tries to parse the .mf.config.json file.

So... what's the ultimate point of this test? Do we really need it?

Reply:
Taken care